### PR TITLE
AB#123411: Fixed the delete reaction method

### PR DESCRIPTION
**Pull request by JoeDavies-6**
Fixes the delete reaction method and the associated reaction list code has been refactored. The commits for database seeding have been moved to the end of the process to ease development.

Includes:
- delete reaction function fixed
- JSDocs added for affected functions
- JS restructured to move into either document head or reaction_list.js
- Surplus print CSS removed
- Database seed commits moved to end of process
- Additional users added to seed_data.yaml
- Export removed from search results

Depends on: 
https://github.com/AI4Green/AI4Green/pull/12

### CODE REVIEW

File: Webapp/sources/blueprints/reaction_list/routes.py
Comment (JoeHeeley2) [inline Webapp/sources/blueprints/reaction_list/routes.py:1]: warning on this file, not sure if its relevant or not

File: Webapp/sources/static/js/reaction_list.js
Comment (JoeHeeley2) [inline Webapp/sources/static/js/reaction_list.js:1]: dom text interpreted as html

**Review by JoeHeeley2 — COMMENTED** [review]
github flagged a couple of errors

File: Webapp/sources/static/js/reaction_list.js
Comment (JoeDavies-6) [reply Webapp/sources/static/js/reaction_list.js:1]: I've added a function now to resolve this warning by interpreting values as text and escaping any special characters (https://github.com/AI4Green/AI4Green/pull/15/files#diff-7d2a5f87072ebfe35371a486852cd22f55fef457292ccef7a1042b05cf3ca31b:~:text=function%20getVal%20(,%7D)

File: Webapp/sources/templates/sketcher_reload.html [hunk 1]
```diff
@@ -83,7 +83,7 @@ <h4>{{ reaction.reaction_id }}</h4>
     <!--Output of reaction image and summary table-->
 
 
-    <div id="dvContainer" class="container-fluid" style="padding-left: 0px">
```
Comment (JoeDavies-6) [inline Webapp/sources/templates/sketcher_reload.html:86]: move scripts in this file into head to be consistent with demo_sketcher.html

**Review by JoeHeeley2 — APPROVED** [review]

**Review by AndyRae — APPROVED** [review]

**Comment by JoeDavies-6** [comment]
> > The commits for database seeding have been moved to the end of the process to ease development.
> 
> Out of curiosity, how come? It'll be fine as seeding was fast in the end for most of these tables, so was likely an over optimisation for now. Except for seeding compounds, which is slow

I changed it because it gave an error during seeding and when I was trying to diagnose and fix this I was getting an error from trying to rerun the seed after the seed had partially ran due to violating unique constraints within the database due to some data already having been committed.

So, thought it may be simpler if the commit happened at the end to prevent partial seeding.

**Pull request merged**